sdlib/d/gc/page.d

```diff
@@ -267,6 +277,7 @@ private:
 		registerBlock(block);
 
 		auto ptr = block.address + n * PageSize;
+		usedPageCount += pages;
```

Comment:
It does not seems smart to me to interleave this in the middle of block operations. Just put it before the reserve call. There is no failure branch anywhere in there.

Reply:
Fixed